packages/OCADml/OCADml.0.5.0/opam

```diff
@@ -0,0 +1,43 @@
+opam-version: "2.0"
+synopsis: "Types and functions for building CAD packages in OCaml"
+description: "Types and functions for building CAD packages in OCaml"
+maintainer: ["Geoff deRosenroll<geoffderosenroll@gmail.com"]
+authors: ["Geoff deRosenroll<geoffderosenroll@gmail.com"]
+license: "GPL-2.0-or-later"
+tags: ["OCADml" "CAD"]
+homepage: "https://github.com/OCADml/OCADml"
+doc: "https://OCADml.github.io/OCADml"
+bug-reports: "https://github.com/OCADml/OCADml/issues"
+depends: [
+  "ocaml" {>= "4.14.0"}
+  "dune" {>= "3.3"}
+  "gg" {>= "1.0.0"}
+  "odoc" {with-doc}
+]
+depopts: [
+  "cairo2" {>= "0.6.2"}
+]
```

Comment:
See https://github.com/ocaml/opam-repository/pull/23653#discussion_r1161926549 for the reasoning:
```suggestion
  "cairo2"
]
conflicts: [
  "cairo2" {< "0.6.2"}
]
```